streaming/java/streaming-runtime/src/main/java/org/ray/streaming/runtime/worker/context/RayRuntimeContext.java

```diff
@@ -50,12 +70,62 @@ public Long getBatchId() {
     return batchId;
   }
 
+  @Override
+  public void setCheckpointId(long checkpointId) {
+    if (this.keyStateManager != null) {
+      this.keyStateManager.setBatchId(checkpointId);
```

Comment:
It looks like that the batchId is an abstracted Id. Comment is required to let developer know why use the batch id to save the checkpoint id